Installation/Updater700100.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace UJM\ExoBundle\Installation;
+
+use Claroline\BundleRecorder\Log\LoggableTrait;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class Updater700100
```

Comment:
Class name doesn't match version 7.0.0. It should be `Updater070000` (or `Updater700000`, if we follow the convention incorrectly used here). Read it as two digits for each part of a semver version (major.minor.patch).